src/material/list/list.md

```diff
@@ -24,18 +24,18 @@ Simple navigation lists can use the `mat-list-item` attribute on anchor tag elem
 
 ```html
 <mat-nav-list>
-   <a mat-list-item href="..." *ngFor="let link of links"> {{ link }} </a>
+  <a mat-list-item href="..." *ngFor="let link of links">{{ link }}</a>
 </mat-nav-list>
 ```
 
-For more complex navigation lists (e.g. with more than one target per item), wrap the anchor 
+For more complex navigation lists (e.g. with more than one target per item), wrap the anchor
 element in an `<mat-list-item>`.
 
 ```html
 <mat-nav-list>
   <mat-list-item *ngFor="let link of links">
-     <a matLine href="...">{{ link }}</a>
-     <button mat-icon-button (click)="showInfo(link)">
+     <a matListItemTitle href="...">{{ link }}</a>
+     <button mat-icon-button (click)="showInfo(link)" matListItemMeta>
```

Comment:
can we add some docs about `matListItemTitle` and `matListItemMeta`? I don't think they had an equivalent in the old version, so they're not documented